doc/api/cli.md

```diff
@@ -1618,6 +1618,7 @@ Node.js options that are allowed are:
 * `--experimental-modules`
 * `--experimental-network-imports`
 * `--experimental-policy`
+* `--experimental-shadow-realm`
```

Comment:
We want to add a section for this flag I assume, between those two https://github.com/nodejs/node/blob/0d58c0be3e1c3013959c02d42a2a2f21dd31c5f8/doc/api/cli.md#L341-L355

Reply:
I was following the flag `--experimental-top-level-await`. It is only added as a valid Node.js option without docs. I think the intention is to prefer `--harmoy-top-level-await` instead of the Node.js one.

Reply:
Quite the contrary, I think the intention is to avoid using V8 flags when we can.